### PR TITLE
feat[added single product-page]

### PR DESCRIPTION
**Pull request by Radheshyam1234**
Single product page

### CODE REVIEW

**Review by lannisterrr — APPROVED** [review]
Good use of axios and react-router to fetch on the basis on product id.

File: src/Components/ProductDetail/ProductDetail.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        <div class="grid-50-50-layout product-detail-card box-shadow">
+          <div class="card-img">
+            <img
+              loading="lazy"
+              src={product.image}
+              alt="Nature"
+              class="img-responsive"
+            />
+          </div>
+          {/* <span class="card-badge text-semibold text-center"> 20% off</span> */}
```
Comment (junaidshaikh-js) [inline src/Components/ProductDetail/ProductDetail.js:31]: remove code comments

**Review by junaidshaikh-js — APPROVED** [review]
Good work! 

Can be merged.

File: src/Components/ProductDetail/ProductDetail.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+            <div class="card-category text-bold">{product.category}</div>
+
+            <div class="card-description">
+              <div class="text-small">
+                <p>{product.name}</p>
+              </div>
+            </div>
+            <div className="card-description">
+              {[1, 2, 3, 4, 5].map((item) => {
```
Comment (anand-tiwari) [inline src/Components/ProductDetail/ProductDetail.js:45]: should create a variable and use that for iteration

File: src/Components/ProductDetail/ProductDetail.js [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+                return (
+                  <i
+                    className="fas fa-star rating-star-icon"
+                    style={{
+                      color: item <= product.rating.starRating ? "#d80230" : "",
+                    }}
+                  ></i>
+                );
+              })}
+              <span className="ml-1">
```
Comment (anand-tiwari) [inline src/Components/ProductDetail/ProductDetail.js:55]: ml-1. class name should be readable

**Review by anand-tiwari — APPROVED** [review]
other codes are fine

**Pull request merged**